go-rest-example/api/platformRoutes.go

```diff
@@ -0,0 +1,60 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/speedrun-website/speedrun-rest/data"
+)
+
+func loadPlatformRoutes(router *mux.Router) {
+	platformRouter := router.PathPrefix("/platforms").Subrouter()
+
+	// Index
+	platformRouter.
+		Path("/").
+		HandlerFunc(getPlatforms).
+		Name("PlatformIndex").
+		Methods("GET")
+
+	// Show
+	platformRouter.
+		Path("/{name}").
```

Comment:
This was pretty silly. Path parameter should be `ID`, `name` should be a query parameter.